frontend/public/components/configmap-and-secret-data.jsx

```diff
@@ -1,14 +1,36 @@
 import * as React from 'react';
 import * as PropTypes from 'prop-types';
-import { Heading } from './utils';
+import { Heading, CopyToClipboard } from './utils';
 
-export const ConfigMapData = ({data}) => {
+const MaskedData = () => {
+  return <React.Fragment>
+    <span className="sr-only">Value hidden</span>
+    <span aria-hidden="true">&bull;&bull;&bull;&bull;&bull;</span>
+  </React.Fragment>;
+};
+
+const Icon = ({showSecret}) => {
+  return showSecret ? <span className="show-values"><i className="fa fa-eye-slash" aria-hidden="true"></i></span> : <span className="hide-values"><i className="fa fa-eye" aria-hidden="true"></i></span>;
+};
+
+export const KeyValueData = ({data, showSecret}) => {
```

Comment:
I'd probably just keep this inside `SecretData`. If you keep the component, remove the `export` and update the name so it's clear it's only for secrets, not config maps.